src/renderers/shared/reconciler/instantiateReactComponent.js

```diff
@@ -122,6 +122,7 @@ function instantiateReactComponent(node) {
   if (__DEV__) {
     instance._isOwnerNecessary = false;
     instance._warnedAboutRefsInRender = false;
+    instance._isServerSideRendered = false;
```

Comment:
Agree, I was thinking about `false` compared to missing declaration but surprisingly I forgot to consider `null` as an option